symbolic-symcache/src/lib.rs

```diff
@@ -111,6 +111,9 @@ use std::convert::TryInto;
 use std::mem;
 use std::ptr;
 
+use camino::Utf8Component;
+use camino::Utf8Path;
+use camino::Utf8PathBuf;
```

Comment:
Rather than using `camino` directly here, could we replace the utilities in `symbolic-common` instead? They also have some (partially commented out) test that should pass.

Reply:
The one problem with that idea is that ideally `SymCacheConverter` would keep its `path_buffer` field so that a new buffer doesn't have to be allocated every time a file's path needs to be normalized.